scripts/skills/special/rf_first_turn_initiative.nut

```diff
@@ -0,0 +1,71 @@
+this.rf_first_turn_initiative <- ::inherit("scripts/skills/skill", {
+	m = {
+		// Public
```

Comment:
Public is rather misleading as these aren't meant to be accessed/changed from outside necessarily. If you're gonna put a comment here I'd suggest renaming it to Config.